spec/index.bs

```diff
@@ -11943,12 +11967,19 @@ GPUQueue includes GPUObjectBase;
 
                 [=Content timeline=] steps:
 
+                1. Let <var data-timeline=content>contentTimeline</var> be the current [=Content timeline=].
                 1. Let |promise| be [=a new promise=].
-                1. Issue the |resolve steps| on the [=Queue timeline=] of |this|.
+                1. Issue the |synchronization steps| on the [=Queue timeline=] of |this|.
                 1. Return |promise|.
             </div>
             <div data-timeline=queue>
-                [=Queue timeline=] |resolve steps|:
+                [=Queue timeline=] |synchronization steps|:
+
+                1. At an **unspecified point now or in the future**, issue the subsequent steps on
```

Comment:
Not sure, but I don't think this needs to be unspecified? mapAsync puts lower and upper bounds on when the resolve can happen. But here I think there's just one point.

> and no later than the next [device timeline](https://gpuweb.github.io/gpuweb/#device-timeline) operation after the [device timeline](https://gpuweb.github.io/gpuweb/#device-timeline) becomes informed of the completion of all currently-enqueued operations (regardless of whether they use this),

Also here should use similar language about "completion of all currently-enqueued operations" I think.

Reply:
Updated, and also moved this step the the device timeline (still monitoring the completion on the queue timeline) because it retrospect it felt weird to have content timeline jumping straight to the queue timeline.